src/formatDate.js

```diff
@@ -8,7 +8,24 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = Array.from(fromFormat).pop();
+  const newFormat = Array.from(toFormat);
```

Comment:
There's no need to create a new array from `toFormat`. You can directly work with `toFormat` as it is already an array.